src/front/js/component/footer.js

```diff
@@ -1,10 +1,45 @@
-import React, { Component } from "react";
+import React from "react";
+import { Link } from "react-router-dom";
 
-export const Footer = () => (
-	<footer className="footer mt-auto py-3 text-center">
-		<p>
-			Made with <i className="fa fa-heart text-danger" /> by{" "}
-			<a href="http://www.4geeksacademy.com">4Geeks Academy</a>
-		</p>
-	</footer>
-);
+export const Footer = () => {
+  return (
+    <>
+      <footer className="page-footer font-small blue pt-4">
+        <div id="links" class="container text-center">
+          <div class="row row-cols-1 row-cols-sm-2 row-cols-md-4">
+            <div>
+              <Link to="/">
+                <h6>Ask to the Community</h6>
+              </Link>
+            </div>
+            <div>
+              <Link to="/">
```

Comment:
I think that the default styling for links is not very pretty, it has that underline and it turns to blue when we hover the mouse on them...

Could you remove those two things? I think it'll look better without them 🖌️